static/panes/ast-view.js

```diff
@@ -131,6 +138,15 @@ Ast.prototype.getDisplayableAst = function (astResult) {
 
 Ast.prototype.showAstResults = function (results) {
     this.astEditor.setValue(results);
+
+    if (!this.isInitialised) {
```

Comment:
Couple of notes:
* Can we rename this to something less generic-sounding. Naming is hard I know, but this is specific to receiving the very first results. It's not like the ast wasn't initialised before this, it was just awaiting initial results. So... maybe `awaitingInitialResults` ?
* Can we explicitly initialise this to `false` in the constructor? I know it's unnecessary, but humour me. I'm still trying to pretend we live in a C++--like language here :grinning: 

(And ditto the other files too)

Reply:
@mattgodbolt Thank you for your review comments :blush: I fixed them.

By the way I do not know what is the convention at this project so I've amended the changes to my previous commit.

Reply:
The usual convention is to just put additional changes: it's often a problem to change "history" :) In general in git, one keeps moving forward by making subsequent changes once one has ever "published" (be it through a `git push` or a PR or whatever).

Thanks!!